sdk/batch/microsoft-azure-batch/src/test/java/com/microsoft/azure/batch/PoolTests.java

```diff
@@ -608,4 +592,77 @@ public void canCRUDPaaSPool() throws Exception {
             }
         }
     }
+
+    @Test
+    public void canCRUDIaaSPoolWithNodeCommunication() throws Exception {
```

Comment:
Can we combine this with another test? I don't think this feature is different enough to justify creating a pool and spinning up VMs separately.

Reply:
Merged this with an existing test.